lib/localparticipant.js

```diff
@@ -563,6 +594,24 @@ class LocalParticipant extends Participant {
  *   value removes any previously set bitrate limit
  */
 
+/**
+ * Options for publishing a {@link LocalTrack}.
+ * @typedef {object} LocalTrackPublishOptions
```

Comment:
there are more fields that goes into `LocalTrackPublishOptions` like name right? Should they be doc'ed  here too?

Reply:
@makarandp0 No, `name` is specified in LocalTrackOptions when you are creating the `LocalTrack`.